ajax/libs/covervid/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "covervid",
+  "filename": "covervid.min.js",
+  "version": "1.0.0",
+  "description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin extension",
```

Comment:
Please modify the `description` to let it as same with upstream.
`"description": "Make your HTML5 video behave like a background cover image with this lightweight Javascript plugin / jQuery extension"`
ref: https://github.com/stefanerickson/covervid/blob/gh-pages/bower.json#L9